libs/auth-api-lib/src/lib/entities/dto/delegation.dto.ts

```diff
@@ -0,0 +1,33 @@
+import { ApiProperty } from '@nestjs/swagger'
+import {
+  IsBoolean,
+  IsString,
+  IsOptional,
+  IsNumber,
+  IsDateString,
+} from 'class-validator'
+// For use with deleting Inserting and updating Delegation
+export class DelegationDTO {
+  @IsString()
+  @ApiProperty()
+  fromNationalId!: string
+  @IsString()
+  @ApiProperty()
+  fromDisplayName!: string
```

Comment:
Do we want the client to provide this? I don't think we have a plan for it now. The plan is to generally show the person's real name from Þjóðskrá. Not sure how/where we query/cache that though. Maybe the graphql layer.

Reply:
Was thinking the client would provide this (to save calls to Þjóðskrá). Could default to the real name when registering, with the option of overriding with any alias (for display in dropdown and on registration page, not used in tokens)?
But your call.

Reply:
Ok, makes sense. Let's keep this around for now. 👍